tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,62 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd alertmanager_ip grafana_ip prometheus_ip
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=beta --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	echo "Wait for all unit agents to be in active condition"
+	wait_for 0 '[.applications[] | select((.units[] | .["juju-status"].current != "idle") and (.units[] | .["workload-status"].current != "error"))] | length' 1800
```

Comment:
This is broken. You've gotten the negation incorrect I think
https://en.wikipedia.org/wiki/De_Morgan%27s_laws
You need to switch the `and` for an `or`, and also negate the second part of the statement (`== "error"` instead of `!= "error"`) since if an application is in error, we want to count it as an application which is not idle

Reply:
I would also extract this into a function in `includes` since it will be useful elsewhere

Reply:
Noted.